app/src/main/java/one/block/androidexampleapp/MainActivity.java

```diff
@@ -93,9 +93,9 @@ public void finish(boolean success, String updateContent) {
                 MainActivity.this.update();
                 btnTransfer.setEnabled(true);
 
-                if (success) {
-                    executeCheckBalance(nodeUrl);
-                }
+//                if (success) {
```

Comment:
Commented out because this isn't relevant to CFA/CFD setup and will fail if your chain isn't setup properly.